src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
Part of my goal here is that the scoring server would not be couples to the h2o version of the model it wraps. So it would actually be beneficial to test with older models. And the test model jars do include their needed versions of hex.genmodel.GenModel, but these are not being used, in favor of the version listed in the pom. So I've been trying to figure out how to use maven scopes to exclude the dependency during testing, but I've had no luck. Do you think this kind of version independence is a viable approach?